features/cli/oc_import_image.feature

```diff
@@ -212,3 +212,46 @@ Feature: oc import-image related feature
       | o               | yaml    |
     Then the expression should be true> @result[:parsed]['status']['imageChangeTriggers'][0]['lastTriggeredImageID'].include? '<%= project.name %>/ruby-26-rhel7'
 
+ # @author wewang@redhat.com
+  # @case_id OCP-45471
+  @azure-ipi @openstack-ipi @baremetal-ipi @vsphere-ipi @gcp-ipi @aws-ipi
+  @azure-upi @aws-upi @openstack-upi @vsphere-upi @gcp-upi
```

Comment:
Please do not copy those tags. We rely on those tags to run tests for 4.9/4.10 on Prow.